dist/angular-chartist.js

```diff
@@ -22,17 +22,15 @@
         this.responsiveOptions = $scope.responsiveOptions() || null;
     }
 
+    AngularChartistCtrl.$inject = ['$scope'];
```

Comment:
I believe this is more maintainable over the `[dep, dep, dep, function]` notation in injectable targets... maybe only my preference?